crates/bevy_math/src/primitives/mod.rs

```diff
@@ -29,3 +29,20 @@ pub enum WindingOrder {
     #[doc(alias("Degenerate", "Collinear"))]
     Invalid,
 }
+
+/// A trait for getting measurements of 2D shapes
+pub trait Measured2d {
+    /// Get the perimeter of the shape
+    fn perimeter(&self) -> f32;
+
+    /// Get the area of the shape
+    fn area(&self) -> f32;
+}
+
+/// A trait for getting measurements of 3D shapes
+pub trait Measured3d {
+    /// Get the surface area of the shape
+    fn area(&self) -> f32;
+    /// Get the volume of the shape
```

Comment:
Giga nitpick: These have different line spacing

Reply:
haha, you're right, it's fixed now :)